cloudflare.install.sh

```diff
@@ -208,11 +208,15 @@ install $SOURCE_DIR/APIKey $INSTALL_DIR/bin/admin/CloudFlare/APIKey
 chmod 0700 $INSTALL_DIR/bin/admin/CloudFlare/APIKey
 echo "mode=simple" > $INSTALL_DIR/bin/admin/CloudFlare/APIKey.conf
 
+# Get PHP Version
+CPANELSUPPORTEDPHPPATH=`ls -al $INSTALL_DIR/3rdparty/bin/php | awk '{print $11}'`
+PHPVERSION=`echo $CPANELSUPPORTEDPHPPATH | rev | cut -d '/' -f 3 | rev`
```

Comment:
@jwineman That doesn't work. In my opinion rev logic is clear enough. I can add comment to be more clear though?